### ISSUE TITLE
"Clear All" button is not resetting filter to default

### ISSUE THREAD

**Issue by figart** [digett-trs/trs#48]
The "Clear All" button in the filter area doesn't seem to be resetting all values to default. It seems to only work on price, acreage, and sort by. It _does_ appear to be changing the filter results, but it's not visibly clearing the filter fields.

**Comment by emontes** [comment]
In the components/searchResults/index.js there is this function
![image](https://user-images.githubusercontent.com/1318017/196676605-487721c9-047d-4b47-8be3-0648d3f48be2.png)

with the refine(items) function it clears Price and Acreage. And with the custom clear it supose to clear County, Sort By, Status and Search for.  That is the part that it's not working.

The custom clear function is as follows:

```const customClear = () => {
    selectInputRef.current.select.clearValue()
    sortInputRef.current.select.clearValue()
    const sarchInput = searchInputRef.current.childNodes[0]
    sarchInput.value = ""
    statusRadioRef.current.childNodes[0].childNodes[0].click()
  }
```

The variable definitions are

line 84
```
const selectInputRef = createRef()
const sortInputRef = createRef()
const searchInputRef = createRef()
const statusRadioRef = createRef()
``` 

My first theory was that it was causing an error because the use of createRef, it cannot be used inside a function and was causing an error, then I've substituted it by useRef (a hook) but it didn't work either.

I will try other alternatives

**Comment by emontes** [comment]
### First semi-solution
I've already gotten it to actually remove the filters when the "clear all" button is pressed. I've also gotten it to clear the "Search For" value.

What I have not been able to achieve is that it cleans the County and Stauts Values (although it does cancel the filters it does not remove the labels).

**Comment by figart** [comment]
Yes, so far so good. Just a note, though: Before any criteria have been selected, or when "Clear All" is clicked, the header underneath should actually say "Featured Properties". It was doing this just fine for a while, but somewhere this functionality got wiped out, it appears.

**Comment by emontes** [comment]
Thank you very much for that hint, it gives me a very good idea to investigate

**Comment by figart** [comment]
According to Enrique, operation is clearing, but not the values in the fields.
